meetings/08-28-17.md

```diff
@@ -0,0 +1,17 @@
+# 08-28-17
+
+### General
+
+- Define novice
+  - Rerequisites for the repo user / contributor
+  - Address users and developers?
+  - Use cases and suggestions for the repository
+- Adding micromezz to repo, still working on legal approval
+  - Transitioning it to kiCAD (currently in upverta)
+- Rafael and Gustavo working with Michael to fix pin shifting issue on mezzanine for stackability
+- Sahaj still witing on proto mezzanine to arrive
+- Akshit, learning to use kiCAD. 
+
+### Actions:
+
+- Potentially adding a template for upverta
```

Comment:
It is not upverta, it is https://upverter.com/

For reference, there is new news: https://blog.upverter.com/2017/08/28/upverter-joins-altium/

Reply:
Hmmm, I don't know how that happened. I actually looked it up and thought I wrote upverter...

Thanks!